flaskbb/templates/macros.html

```diff
@@ -320,13 +320,15 @@
 </li>
 {% endmacro %}
 
-{% macro render_pagination(page_obj, url, ul_class='') %}
+{% macro render_pagination(page_obj, url, ul_class='', sort_by=None, asc=True) %}
 <ul class='{%- if ul_class -%}{{ ul_class }}{%- else -%}pagination{%- endif -%}'>
+    {% set ordering = 'asc' if asc == True else 'desc' %}
+    {% set sorting = '&sort_by='+(sort_by|urlencode)+'&order_by='+ordering if sort_by is string else '' %}
```

Comment:
Nice, I didn't know about these builtin tests:
http://jinja.pocoo.org/docs/dev/templates/#list-of-builtin-tests